plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -244,7 +250,7 @@ static void ffmpeg_source_open(struct ffmpeg_source *s)
 				s->input, s->input_format,
 				s->buffering_mb * 1024 * 1024,
 				s, get_frame, get_audio, media_stopped,
-				preload_frame, s->is_hw_decoding, s->range);
+				preload_frame, s->is_hw_decoding, s->is_not_seekable, s->range);
```

Comment:
Style:  The line length limit is a strongly preferred 80 columns.  Please break this line accordingly (between `s->is_hw_decoding,` and `s->is_not_seekable`).